src/electron/views/automated-checks/automated-checks-view.tsx

```diff
@@ -68,7 +69,8 @@ export class AutomatedChecksView extends React.Component<AutomatedChecksViewProp
                         </main>
                     </div>
                     <div className={screenshotView}>
```

Comment:
The structure here is interesting! if  `this.props.unifiedScanResultStoreData` is undefined (acording to line 127 on the changed file) we'll get something like
```jsx
<div ...>
   {null}
</div>
```
Is that on purpose?

Reply:
I changed the renderScreenshotView method at some point Friday; is the current version (on the most recent commit) in line with the ideal structure?

Reply:
@smoralesd